src/main/java/com/microsoft/azure/AzureVMManagementServiceDelegate.java

```diff
@@ -1069,13 +1049,6 @@ private static String getLocationName(String location) {
                 return errors;
             }
 
-            //verify password
-            validationResult = verifyAdminPassword(adminPassword);
-            addValidationResultIfFailed(validationResult, errors);
-            if (returnOnSingleError && errors.size() > 0) {
-                return errors;
-            }
-
```

Comment:
This should probably be added back in.  I removed this for bringup but IIRC the deployment will fail if the password is too weak. It just needs to look up the password first.